kolibri/plugins/learn/assets/src/views/content-cards/content-card.vue

```diff
@@ -26,44 +26,62 @@
       <h3 class="card-title">{{ title }}</h3>
       <h4 class="card-subtitle"></h4>
     </div>
+
   </router-link>
 
 </template>
 
 
 <script>
 
+  const CoreConstants = require('kolibri.coreVue.vuex.constants');
+  const values = require('lodash/values');
+  const responsiveWindow = require('kolibri.coreVue.mixins.responsiveWindow');
   const validateLinkObject = require('kolibri.utils.validateLinkObject');
 
   module.exports = {
+    mixins: [responsiveWindow],
+    components: {
+      'content-icon': require('kolibri.coreVue.components.contentIcon'),
+      'progress-icon': require('kolibri.coreVue.components.progressIcon'),
+    },
     props: {
       title: {
         type: String,
         required: true,
       },
-      subtitle: {
```

Comment:
It wasn't being used anywhere in the card, figured it was deprecated. I'm seeing an h4 for it, I can re-add and include it in the template